resource-thief.c

```diff
@@ -0,0 +1,361 @@
+#define _POSIX_C_SOURCE 200809L
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/types.h>
+#include <signal.h>
+#include <fcntl.h>
+
+#include <gbm.h>
+#include <pool-buffer.h>
+#include <wayland-client-protocol.h>
+#include <linux-dmabuf-unstable-v1-client-protocol.h>
```

Comment:
This is a local header, so should be included with quotes.

Reply:
Done; "pool-buffer.h" was also local.